ods_ci/tests/Resources/Files/llm/model_expected_responses.json

```diff
@@ -21,9 +21,13 @@
                     "response_tokens":  7,
                     "response_text": "212 ° f",
                     "streamed_response_text":   "{  'details': {    'input_token_count': '8'  }}{  'tokens': [    {      'text': '▁',      'logprob': -1.2747352123260498    }  ],  'details': {    'generated_tokens': 1  }}{  'generated_text': '212',  'tokens': [    {      'text': '212',      'logprob': -1.0382558107376099    }  ],  'details': {    'generated_tokens': 2  }}{  'generated_text': ' ',  'tokens': [    {      'text': '▁',      'logprob': -0.8835393786430359    }  ],  'details': {    'generated_tokens': 3  }}{  'generated_text': '°',  'tokens': [    {      'text': '°',      'logprob': -0.6830151677131653    }  ],  'details': {    'generated_tokens': 4  }}{  'generated_text': ' ',  'tokens': [    {      'text': '▁',      'logprob': -0.8991543650627136    }  ],  'details': {    'generated_tokens': 5  }}{  'generated_text': 'f',  'tokens': [    {      'text': 'f',      'logprob': -0.44153422117233276    }  ],  'details': {    'generated_tokens': 6  }}{  'tokens': [    {      'text': '\u003c/s\u003e',      'logprob': -0.015602776780724525    }  ],  'details': {    'finish_reason': 'EOS_TOKEN',    'generated_tokens': 7  }}"
+                },
+                "mpt-7b-instruct2": {
+                    "response_tokens": 20,
+                    "response_text": "\nWater boils at 100 degrees Celsius (or 212 degrees Fahrenheit).\nWhat",
+                    "streamed_response_text": "{    'inputTokenCount': 10}{    'generatedTokenCount': 2,    'text': '\n'}{    'generatedTokenCount': 3,    'text': 'I a'}{    'generatedTokenCount': 4,    'text': 'm no'}{    'generatedTokenCount': 5,    'text': 't sur'}{    'generatedTokenCount': 6,    'text': 'e i'}{    'generatedTokenCount': 7,    'text': 'f thi'}{    'generatedTokenCount': 8,    'text': 's i'}{    'generatedTokenCount': 9,    'text': 's th'}{    'generatedTokenCount': 10,    'text': 'e righ'}{    'generatedTokenCount': 11,    'text': 't plac'}{    'generatedTokenCount': 12,    'text': 'e t'}{    'generatedTokenCount': 13,    'text': 'o as'}{    'generatedTokenCount': 14,    'text': 'k thi'}{    'generatedTokenCount': 15,    'text': 's questio'}{    'generatedTokenCount': 16,    'text': 'n'}{    'generatedTokenCount': 17,    'text': ', bu'}{    'generatedTokenCount': 18,    'text': 't '}{    'generatedTokenCount': 19,    'text': 'I a'}{    'generatedTokenCount': 20,    'text': 'm trying',    'stopReason': 'MAX_TOKENS'}"
```

Comment:
move it at the end of the list, otherwise query index breaks in other tests

Reply:
It's not a new query and it's already at the end of the list